src/components/BottomSheet/BottomSheet.js

```diff
@@ -4,6 +4,7 @@ import {StyleSheet, View} from 'react-native';
 import RBSheet from 'react-native-raw-bottom-sheet';
 
 import {colors} from '../../utils/colors';
+import dimen from '../../utils/dimen';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `dimen` is new in this hunk. Ensure that the `dimen` module exists and is properly exported from the `../../utils/dimen` path. If it doesn't exist, it will cause a runtime error.
```diff
+ import dimen from '../../utils/dimen';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->